deploy/cloudbuild-release-lts.yaml

```diff
@@ -49,6 +49,7 @@ steps:
     - 'make'
     - 'release-lts'
     - 'VERSION=$TAG_NAME'
+    - 'SCANNING_MARKER=$_SCANNING_MARKER'
```

Comment:
It can be defined in cloud build trigger, i just did it thanks!
![Screenshot 2023-06-01 at 9 20 52 AM](https://github.com/GoogleContainerTools/skaffold/assets/102683393/3046649a-77b4-4a1e-b5cf-301bf6ada642)